packages/statemanager/test/proofStateManager.spec.ts

```diff
@@ -132,7 +147,6 @@ describe('ProofStateManager', () => {
       await trie._db.put(key, bufferData)
     }
     trie.root(stateRoot!)
-    await stateManager.putAccount(address, new Account())
```

Comment:
This had to be deleted because this actually creates an empty account in the trie (thus changing the state root) - this test has been wrong the entire time, it seems :thinking: